qualtran/bloqs/rotations/phase_gradient.py

```diff
@@ -67,6 +69,21 @@ class PhaseGradientUnitary(GateWithRegisters):
 
     Registers:
         phase_grad: A THRU register which the phase gradient is applied to.
+        (optional) ctrl: A THRU register which specifies the control for this gate. Must have
+            `is_controlled` set to `True` to use this register.
+
+    Arguments:
+        bitsize: The number of qubits of the register being acted on
+        exponent: $t$ in the above expression for $\omega_{n, t}$, a multiplicative factor
+            for the phases applied to each state. Defaults to 1.0.
+        is_controlled: `bool` which determines if the unitary is controlled via a `ctrl` register.
+        eps: The precision for the underlying `ZPowGates`.
+
+    Costs:
+        qubits: 0 ancilla qubits are allocated.
+        T-gates: Only uses 1 T gate explicitly but does rely on more costly Z rotations.
+        rotations: Uses $n$ rotations of varying angles. The lowest accuracy rotation is
+            a single T gate and the most accurate rotation has an angle of $1/2^n$.
```

Comment:
Replace `accuracy` with `rotation angle` ?

Reply:
made the sentence symmetric so it makes more sense.